### PR TITLE
Avivash/58 improve homepage UI(does not include 404 or User Settings)

### PR DESCRIPTION
**Pull request by avivash**
**Note: 404 Page sub-branch was merged here: https://github.com/webnative-examples/webnative-app-template/pull/85 and User Settings sub-branch was merged here https://github.com/webnative-examples/webnative-app-template/pull/84**

# Description

- Implementing the new UI refresh for all existing routes(User Settings and 404 routes will be addressed different PRs)
- These changes look like a lot, but there are only a couple new components. Most of the other changes are style tweaks to existing files

## Link to issue

https://github.com/webnative-examples/webnative-app-template/issues/58

## Type of change

- [x] New feature (non-breaking change that adds functionality)
- [x] Refactor (non-breaking change that updates existing functionality)
- [x] Comments have been added/updated

## Screenshots/Screencaps

https://www.loom.com/share/7c5b9b3048eb4b07b7418cfe798e2ff4

### CODE REVIEW

**Comment by avivash** [comment]
@bgins @jessmartin there's probably a good argument to be made that I should have merged this PR first, but oh well 🤷🏼 next time!

File: src/components/auth/delegate-account/ConnectBackupDevice.svelte [hunk 1]
```diff
@@ -2,39 +2,48 @@
...
+  import { addNotification } from '$lib/notifications'
 
-  export let qrcode
+  export let qrcode: HTMLOrSVGElement
   export let connectionLink: string
   export let backupCreated: boolean
 
   const copyLink = async () => {
     await clipboardCopy(connectionLink)
+    addNotification('Copied to clipboard', 'success')
```
Comment (bgins) [inline src/components/auth/delegate-account/ConnectBackupDevice.svelte:14]: Yeah, good idea to add a notification for this. 👍

File: src/components/auth/delegate-account/DelegateAccount.svelte [hunk 2]
```diff
@@ -10,8 +10,16 @@
     dispatch('cancel')
   }
 
+  /**
+   * Auto submit the form when the pinInput is equal to the TARGET_PIN_LENGTH
```
Comment (bgins) [inline src/components/auth/delegate-account/DelegateAccount.svelte:14]: Excellent! Great this got implemented with this work. 🕺

File: src/components/home/Authed.svelte [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      storage in your app. See it in action with our photo gallery demo.
+    </p>
+    <a class="btn btn-primary" href="/gallery">Try the Photo Gallery Demo</a>
+  </div>
+
+  <div class="flex flex-col items-start justify-center gap-5">
+    <h2 class="text-lg">Device Connection Demo</h2>
+    <p>
+      With Webnative SDK, a user’s account lives only on their connected
+      devices—entirely under their control. It’s easy for them to connect as
```
Comment (bgins) [inline src/components/home/Authed.svelte:23]: Should there be spaces around the `—` dash?

File: src/components/home/Authed.svelte [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      many devices as they’d like. For recoverability, we recommend they always
+      connect at least 2.
```
Comment (bgins) [inline src/components/home/Authed.svelte:25]:
```suggestion
      connect at least two.
```

Copy editing nitpick. Numbers less than ten (and sometimes more) should be spelled out. Reference: https://www.grammarly.com/blog/when-to-spell-out-numbers/

File: src/components/home/Public.svelte [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    </p>
+
+    <ul class="mb-6 pl-6 list-disc">
+      <li>
+        <span class="font-bold">modern, passwordless accounts</span>
+        , without a complex and costly cloud-native back-end
+      </li>
+      <li>
+        <span class="font-bold">user-controlled data</span>
+        , secured by default via our encrypted-at-rest file storage protocol
```
Comment (bgins) [inline src/components/home/Public.svelte:23]:
```suggestion
        , secured by default with our encrypted-at-rest file storage protocol
```

Suggested edit. "via" is less accessible to ESL readers than "with".

File: src/components/Header.svelte [hunk 6]
```diff
@@ -1,53 +1,93 @@
...
-    <Brand />
-    <span class="text-xl ml-2 hidden md:inline">{appName}</span>
+<header class="navbar flex bg-base-100 pt-4">
+  <div class="lg:hidden">
+    {#if $sessionStore.authed}
+      <label
+        for="sidebar-nav"
+        class="drawer-button cursor-pointer -translate-x-2"
+      >
+        <Hamburger />
```
[the remark this answers is not in the document]
Comment (bgins) [reply src/components/Header.svelte:29]: Ah nevermind! Missed that this only shows up on narrower interfaces. 👌

**Review by bgins — APPROVED** [review]
Fantastic! This new version is looking great and works well. 🤩 🎉 

I've requested a couple of minor changes, some of them with me putting on my technical writing hat. I realize these copy edits fall a bit outside of the PR scope, but I think they are worth considering nonetheless.

File: src/components/Footer.svelte [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  import { themeStore } from '../stores'
+
+  /**
+   * I typically don't do browser specific tweaks, but Firefox doesn't fully support background-clip
+   * so the marquee text is cut off half way through the animation
+   */
+  const isFirefox = navigator.userAgent.indexOf('Firefox') != -1
```
[the remark this answers is not in the document]
Comment (avivash) [reply src/components/Footer.svelte:8]: oops 😬 thanks 👍🏼

File: src/components/home/Authed.svelte (hunk 3 above)
Comment (avivash) [reply src/components/home/Authed.svelte:23]: probably, will update!

Comment (avivash) [reply src/components/home/Authed.svelte]: updated!

File: src/components/Footer.svelte [hunk 8]
```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  import { themeStore } from '../stores'
+
+  /**
+   * I typically don't do browser specific tweaks, but Firefox doesn't fully support background-clip
```
[the remark this answers is not in the document]
Comment (avivash) [reply src/components/Footer.svelte]: updated!

File: src/components/Footer.svelte (hunk 7 above)
[the remark this answers is not in the document]
Comment (avivash) [reply src/components/Footer.svelte]: fixed!

File: src/components/home/Public.svelte (hunk 5 above)
Comment (avivash) [reply src/components/home/Public.svelte]: updated!

File: src/components/home/Authed.svelte (hunk 4 above)
Comment (avivash) [reply src/components/home/Authed.svelte]: updated!

**Pull request merged**